fenics/environment.yml

```diff
@@ -0,0 +1,6 @@
+channels:
+- conda-forge
+dependencies:
+  - fenics =2019.1.0
+  - mshr =2019.1.0
+  - seaborn
```

Comment:
```suggestion
```
This should be moved to `pyiron/base`.

Reply:
Done! There were changes needed to `base/environment.yml`, `damask/environment.yml`, and `fenics/environment.yml`.